pyomo/contrib/doe/doe.py

```diff
@@ -1663,12 +1846,13 @@ def compute_FIM_full_factorial(
             - keys of model's experiment_inputs
             - "log10 D-opt": list of log10(D-optimality)
             - "log10 A-opt": list of log10(A-optimality)
+            - "log10 pseudo A-opt": list of log10(trace(FIM))
             - "log10 E-opt": list of log10(E-optimality)
             - "log10 ME-opt": list of log10(ME-optimality)
             - "eigval_min": list of minimum eigenvalues
             - "eigval_max": list of maximum eigenvalues
             - "det_FIM": list of determinants
-            - "trace_FIM": list of traces
+            - "trace_cov": list of traces
```

Comment:
Here you have replaced `trace_FIM` with `trace_cov` but down on line 1911, when setting up the `fim_factorial_results` object you have appended `trace_cov` so the dictionary contains both which seems inconsistent. Did you mean to include both or just one of `trace_FIM` and `trace_cov`?